components/dependency_injection/tags.rst

```diff
@@ -133,11 +133,11 @@ custom tag::
     {
         public function process(ContainerBuilder $container)
         {
-            if (!$container->hasDefinition('acme_mailer.transport_chain')) {
+            if (!$container->has('acme_mailer.transport_chain')) {
                 return;
             }
 
-            $definition = $container->getDefinition(
+            $definition = $container->get(
```

Comment:
You shouldn't use `get()` in a compiler pass (the definition may be modified by other passes). Use `findDefinition()` instead.